src/components/dashboard/FeedList.js

```diff
@@ -33,16 +33,6 @@ export type FeedListProps = {
   listFooterComponent: React.ReactNode,
 }
 
-const getItemLayout = (_: any, index: number) => {
```

Comment:
Why this removed ? Didn’t it break design or something else ?

Reply:
getItemLayout is used here and should be used, if we have static size of items. Now we have dynamic size of items (news) and getItemLayout won't work right